### PR TITLE
[Form][Console] Use dumper

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Continuation of #28898 for `debug:form`

### CODE REVIEW

File: src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php [hunk 1]
```diff
@@ -183,20 +183,16 @@ private function normalizeAndSortOptionsColumns(array $options)
...
+        }
 
-            return [
-                $prefix.'file' => $a[$prefix.'file'],
-                $prefix.'line' => $a[$prefix.'line'],
-            ];
-        }]);
-        $dumper = new CliDumper(null, null, CliDumper::DUMP_LIGHT_ARRAY | CliDumper::DUMP_COMMA_SEPARATOR);
-        $dumper->setColors($this->output->isDecorated());
+        // BC symfony/console <4.3
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php]: it's not BC at this point, because nothing has been deprecated

Comment (ro0NL) [reply src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php]: what about "preserve compatibility with symfony/console <4.3."?

Comment (nicolas-grekas) [reply src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php]: what about bumping by adding a conflict symfony/console <4.3 instead?

Comment (nicolas-grekas) [reply src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php]: Actually that wouldn't change much the code here :)

Comment (nicolas-grekas) [reply src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php]: I would just remove the comment :)

Comment (ro0NL) [reply src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php]: wasnt sure about the conflict rule :) now fixed.

Comment (ro0NL) [reply src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php]: and missed your comments ..., i think always using the Console dumper is the ideal scenario (and now doing the same in #27684 )

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request merged**